Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
```

Comment:
### [guard](http://guardgem.org)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 1.4.0 → 2.18.0

[_~~change-log~~, [source-code](https://github.com/guard/guard), [gem-diff](https://my.diffend.io/gems/guard/1.4.0/2.18.0)_]





<details>
<summary>Commits</summary>

A change of **1218** commits. See the full changes on [the compare page](https://github.com/guard/guard/compare/v1.4.0...v2.18.0).

These are the first 10 commits:
- (4cf68a2) [Merge pull request #749 from pjump/autoreload](https://github.com/guard/guard/commit/4cf68a2b814201b7cfb7df382a04ec599c763620)
- (2817f00) [Release 2.12.4](https://github.com/guard/guard/commit/2817f00810fdd841150350e7235d4bf2e56f855c)
- (07f6b13) [fix RuboCop offense](https://github.com/guard/guard/commit/07f6b13ad135ee49f0027e0f982c20294054c605)
- (f06f9b2) [detect used plugins more accurately](https://github.com/guard/guard/commit/f06f9b2921982e23c0c8c722b9734fbd7fbbf543)
- (3fbd489) [improve error message when template fails](https://github.com/guard/guard/commit/3fbd4897ad482c7f0aeb46454acf3e428b8b1ccb)
- (54482b7) [add TODOs](https://github.com/guard/guard/commit/54482b758f346f37f3f8fa10bc5646f884725a8b)
- (09e3f68) [update RuboCop in dev](https://github.com/guard/guard/commit/09e3f68c9852fad7e4e10563958364e59776f9d6)
- (47d7c15) [fix RuboCop offenses and update styleguide](https://github.com/guard/guard/commit/47d7c1506d95490cebaab402f26a8acc6306bbdc)
- (4590bf2) [Merge pull request #752 from guard/improve_plugin_handling](https://github.com/guard/guard/commit/4590bf281e9c3e746b29649b274ee15d54d75ddb)
- (10ea1da) [Release 2.12.5](https://github.com/guard/guard/commit/10ea1da68d1142227370de2feb061a0d0a8d3200)


</details>